Makefile

```diff
@@ -317,6 +317,33 @@ clean-volumes:
 compose:
 	$(foreach MODULE, $(CANDIG_MODULES), $(MAKE) compose-$(MODULE);)
 
+#-- Temp --
+compose-authz:
+	docker-compose -f $(DIR)/lib/keycloak/docker-compose.yml up -d 2>&1
```

Comment:
Would there be a need to start these services individually? If not, it is recommended to combine these under $(DIR)/lib/auth/docker-compose.yml